hibernate-core/src/main/java/org/hibernate/sql/results/spi/ScrollableResultsConsumer.java

```diff
@@ -27,7 +27,8 @@ public class ScrollableResultsConsumer<R> implements ResultsConsumer<ScrollableR
 	/**
 	 * Singleton access to the standard scrollable-results consumer instance
 	 */
-	public static final ScrollableResultsConsumer INSTANCE = new ScrollableResultsConsumer();
+	@SuppressWarnings( "rawtypes" )
+	private static final ScrollableResultsConsumer INSTANCE = new ScrollableResultsConsumer();
```

Comment:
Please revert and deprecate for removal instead. This is an SPI, so we'd have to first deprecate the field.